demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -630,6 +631,17 @@ def validate_added_files(self, added_files, modified_files):
                                                          added_files=added_files))
         return all(valid_files)
 
+    @staticmethod
+    def should_raise_pack_version(pack: str) -> bool:
+        """
+        Args:
+            pack: The pack name.
+
+        Returns: False if pack is in IGNORED_PACK_NAMES else True.
+
+        """
+        return pack not in IGNORED_PACK_NAMES
```

Comment:
what about `ApiModules` ?

Reply:
I added it to IGNORED_PACK_NAMES as requested

Reply:
i think you forgot to push it then

Reply:
yeah, just noticed. Pushed it now